Cargo.toml

```diff
@@ -11,6 +11,7 @@ repository = "https://github.com/sorear/smetamath-rs"
 clap = "2.5.2"
 filetime = "0.1.10"
 fnv = "1.0.2"
+regex = "0.1.71"
```

Comment:
That is a bit alarming, but regex is just too useful to stay away from for long, so I think it is worth inclusion even if it seems like overkill right now. SMM3 is nowhere near feature complete yet so some growth in the binary from the current baseline is expected. (Don't suppose there is an alternate regex crate...)